linchpin/cli/__init__.py

```diff
@@ -245,31 +249,30 @@ def lp_down(self, pinfile, targets='all'):
         pass
 
 
-    def find_topology(self, topology):
+    def find_include(self, filename, ftype='topology'):
         """
-        Find the topology to be acted upon. This could be pulled from a
-        registry.
+        Find the included file to be acted upon.
 
         :param topology:
             name of topology from PinFile to be loaded
```

Comment:
docs need updating.

Reply:
This has been updated in the code, and will be submitted in the next PR.